genon/tools/genos_tools/viewer/docling_viewer.py

```
@@ -0,0 +1,255 @@
+    
+    # 🚀 pages_content 존재 여부 체크
+    if not viewer.pages_content: 
+        print("❌ 표시할 페이지 데이터가 없습니다. (JSON 구조나 load_data 로직 확인 필요)")
+        return
+
+    class ReusableServer(socketserver.TCPServer):
+        allow_reuse_address = True
+
+    server = ReusableServer(("", 8000), DoclingHandler)
+    server.viewer = viewer
+    print(f"🚀 Docling 통합 뷰어 서버 시작: http://localhost:8000")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)